symfony/plugins/orangehrmMarketPlacePlugin/lib/manager/APIManagerService.php

```diff
@@ -30,6 +30,10 @@ class APIManagerService
      * url for get addons from marketplace
      */
     const ADDON_LIST = '/api/v1/addon';
+    /**
+     * url to get paid installation pending addons from marketplace
+     */
+    const PAID_INSTALLATION_PENDING_ADDONS = '/api/v1/getPaidInstallationPendingAddons';
```

Comment:
This is against rest concept..  dont add the endpoint for custom things 

Change '/api/v1/addon' endpoint to except paramas 
```
/api/v1/addon?status='Paid'&install='pending'
```
@Damith88 don't add custome endpoints

Reply:
If we are going to do this we may need to revisit the database design and APIs in the marketplace completely. Currently we are having API endpoints, services and repositories for almost each and every entity in the marketplace which I also think should not.

Reply:
We shouldn't introduce custom endpoints .. If there is an issue with old db structure we should change it for good.

Reply:
changed the url as following
/api/v1/addon-payment-status